src/test/addrman_tests.cpp

```diff
@@ -296,23 +296,23 @@ BOOST_AUTO_TEST_CASE(addrman_find)
     addrman.Add(addr2, source2);
     addrman.Add(addr3, source1);
 
-    // Test 17: ensure Find returns an IP matching what we searched on.
+    // Test: ensure Find returns an IP matching what we searched on.
     CAddrInfo* info1 = addrman.Find(addr1);
     BOOST_CHECK(info1);
     if (info1)
```

Comment:
That makes sense. Still trying to figure this out. @jnewbery, what's the coding standard that I should change the if to?